examples/puppetmaster/test.sh

```diff
@@ -31,14 +32,29 @@ cleanup() {
 
 main() {
   cleanup
-  if [ "$CLEAN_UP_ON_EXIT" = true ]; then
+  if [[ "${CLEAN_UP_ON_EXIT}" = true ]]; then
     trap cleanup EXIT
   fi
 
   start_services
   setup_conjur
   wait_for_puppetmaster
-  install_required_module_dependency
+
+  if [[ "${INSTALL_PACKAGED_MODULE}" = true ]]; then
+    # This branch is generally exercised for testing production builds. It installs the
+    # packaged module from the mounted onto directory '/conjur/pkg'.
+
+    install_conjur_module
+  else
+    # This brach is generally exercised during development. It symlinks the source from
+    # mounted onto directory '/conjur' to
+    # '/etc/puppetlabs/code/environments/production/modules/conjur', and installs any
+    # module dependencies.
+
+    symlink_conjur_module
+    install_required_module_dependency
+  fi
```

Comment:
This installation block probably should be pulled out into a method of its own.

Reply:
I thought so as well initially but kept it this way because I think it makes it clear in main that there's branching. I think it's more readable this way.

Reply:
Fair. We can leave this for now though it does seem a bit easy to group as an "install_module" function.